translations/all_map_locations.tex/es.po

```diff
@@ -369,38 +291,20 @@ msgid ""
 "}\n"
 "\n"
 msgstr ""
+"{2}\n"
 "{\\centering\n"
-"  \\phantom{j}\\textbf{Artefacto}\\\\\n"
-"  \\framedimage[\\linewidth]{\\map_locations/artifact_symbol.jpg}\n"
-"\n"
-"  \\bigskip\n"
-"\n"
-"  \\phantom{j}\\textbf{Recurso}\\\\\n"
-"  \\framedimage[\\linewidth]{\\map_locations/resource_symbol.jpg}\\\\\n"
-"\n"
-"  \\bigskip\n"
-"\n"
-"  \\phantom{j}\\textbf{Tesoro}\\\\\n"
-"  \\framedimage[\\linewidth]{\\map_locations/treasure_symbol.jpg}\\\\\n"
-"\n"
-"  \\bigskip\n"
 "\n"
 "  \\phantom{j}\\textbf{Fuente de la Juventud}\\\\\n"
 "  \\framedimage[\\linewidth]{\\map_locations/fountain_of_youth.jpg}\\\\\n"
 "\n"
-"  \\filbreak\n"
-"\n"
-"  \\phantom{j}\\textbf{Molino}\\\\\n"
-"  \\framedimage[\\linewidth]{\\map_locations/windmill.jpg}\\\\\n"
-"\n"
 "  \\bigskip\n"
 "\n"
 "  \\phantom{j}\\textbf{Tumba del Guerrero}\\\\\n"
 "  \\framedimage[\\linewidth]{\\map_locations/warriors_tomb.jpg}\\\\\n"
 "\n"
 "  \\bigskip\n"
 "\n"
-"  \\phantom{j}\\textbf{Jardín Místico}\\\\\n"
+"  \\textbf{Jardín Místico}\\\\\n"
```

Comment:
```suggestion
"  \\phantom{j}\\\\textbf{Jardín Místico}\\phantom{j}\\\\\\\n"
```
Return it, as it's breaking the layout. Look at the picture now - it's not aligned with the picture next to it.